src/pushNotifier.ts

```diff
@@ -24,15 +24,29 @@ export class PushNotifier {
 		this.url = url;
 	}
 
-	notify({ title = "cross-seed", body, ...rest }: PushNotification): void {
+	async notify({
+		title = "cross-seed",
+		body,
+		...rest
+	}: PushNotification): Promise<void> {
 		if (this.url) {
-			fetch(this.url, {
-				method: "POST",
-				headers: { "Content-Type": "application/json" },
-				body: JSON.stringify({ title, body, ...rest }),
-			}).catch(() => {
-				logger.error({ message: "Failed to send push notification" });
-			});
+			try {
+				const response = await fetch(this.url, {
+					method: "POST",
+					headers: { "Content-Type": "application/json" },
```

Comment:
mind sticking a `User-Agent` header here while we're at it? If you search through the codebase for `User-Agent` you should be able to find the standard user agent